src/main/java/mate/academy/hibernate/relations/service/impl/CountryServiceImpl.java

```diff
@@ -1,16 +1,25 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import java.util.Optional;
+import mate.academy.hibernate.relations.dao.impl.CountryDaoImpl;
 import mate.academy.hibernate.relations.model.Country;
 import mate.academy.hibernate.relations.service.CountryService;
 
 public class CountryServiceImpl implements CountryService {
+    private CountryDaoImpl countryDao;
+
+    public CountryServiceImpl(CountryDaoImpl countryDao) {
+        this.countryDao = countryDao;
+    }
+
     @Override
     public Country add(Country country) {
-        return null;
+        return countryDao.add(country);
     }
 
     @Override
     public Country get(Long id) {
-        return null;
+        Optional<Country> optionalCountry = countryDao.get(id);
+        return optionalCountry.get();
```

Comment:
Directly calling `get()` on an `Optional` without checking if it's present can lead to `NoSuchElementException`. It's better to provide a way to handle the case when the country with the given id does not exist, either by throwing a custom exception with a clear message or by returning `null`.